solutions/common-shared-flows/error-handling-v1/sharedflowbundle/policies/JS.PopulateDefaultErrorVariables.xml

```diff
@@ -0,0 +1,3 @@
+<Javascript timeLimit="500" name="JS.PopulateDefaultErrorVariables">
```

Comment:
Looking at the JS, I can't think of a scenario which would warrant increasing the default `timeLimit` of 200ms to 500ms. If anything, I'd reduce it from the default, but probably best to not go down that path. Thoughts?